public/assets/styles/_sections/Support.less

```diff
@@ -0,0 +1,80 @@
+#Support {
+  .mainImage {
+    background-image: url('../images/Support_image_min.jpg');
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+    width: 120%;
+    margin-left: -11%;
+    img {
+      opacity: 0;
```

Comment:
Seems like this is broken. Because of this, we are not seeing any image until after the 768 breakpoint